packages/medusa/src/api/routes/admin/draft-orders/create-draft-order.js

```diff
@@ -0,0 +1,65 @@
+import { MedusaError, Validator } from "medusa-core-utils"
+import { defaultFields, defaultRelations } from "."
+
+export default async (req, res) => {
+  const schema = Validator.object().keys({
+    status: Validator.string()
+      .valid("open", "awaiting", "completed")
+      .optional(),
+    email: Validator.string()
+      .email()
+      .required(),
+    billing_address: Validator.address().optional(),
+    shipping_address: Validator.address().optional(),
+    billing_address_id: Validator.string().optional(),
+    shipping_address_id: Validator.string().optional(),
```

Comment:
This would require us to change the implementation of `CartService.create` to handle nested id's in addresses or alternatively make an unpretty solution in `DraftOrderService.create`, in which we transform `billing_address.id` to `billing_address_id` before calling the `CartService`.

Reply:
Could we maybe just do it in the controller so that the API is a bit cleaner.

```js
if (shipping_address.id) {
  toCreate.shipping_address_id = shipping_address.id
} else {
  // Use individual fields.
}
```

Reply:
Yes... This is an option. We should prioritize clean request body over clean controller impl., so I will add this.